DRAFT BLOG - how to make your startup transparent (#3852)
PostHog/posthog.com: contents/blog/how-to-run-transparent-company.md

---
date: 2022-07-26
title: "How to run a transparent startup"
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
categories: ["CEO diaries", "Inside PostHog"]
author: ["james-hawkins"]
featuredImage: ../images/blog/posthog-ceo-diary-blog.png
featuredImageType: full
---

> There really hasn't been a single situation where sharing pretty much everything we do hasn't been to our company's advantage.

Reviewer: ```suggestion
There really hasn't been a single situation where transparency hasn't been to our company's advantage.
```

Just cutting out some wordy qualifiers to simplify the read.

> Most companies seem to share when necessary but not by default. I hope this piece changes that - as a startup, you've done something wrong if you don't trust your own team.

Reviewer 2: ```suggestion
Most companies seem to share when necessary but not by default. I hope this piece changes that because, as a startup, you've done something wrong if you don't trust your own team.
```

> Being realistic, to help this most people - this is mainly written about _internal_ transparency... and it's startup-focussed too, since that's what I know.

Reviewer: I don't understand some of how this is expressed. I think I'd probably reword this to something like:

```suggestion
It's worth noting that these are my views on how and why more startups should offer greater _internal_ transparency. I've focused on internal transparency because I believe it can be realistically practiced and helps the most people. I've written about startups because that's what I know.
```

It's wordier, I know. :(

> If you want to gain even more advantages here, I would encourage you though to share this stuff by default externally unless there's a good reason not to.

Reviewer: ```suggestion
If you want to gain even more of an advantage, I would encourage you to share this stuff externally by default unless there's a good reason not to.
```

There are probably some startups where this doesn't work. Perhaps in design led companies, where control might be a competitive advantage and is more important than autonomy, or in companies that aren't managed consistently - you can't share a non-existent pay structure, for example (and, yes, it is hard work to implement something like that from very early, but we felt that work was worth it to prevent unfairness and employee churn).

> ## Why this matters

Reviewer: ```suggestion
## Why transparency matters
```

Reading on from the previous sentence makes this feel unclear. Are we talking about pay structures in particular, or transparency in general.

This section is a bit effusive. Skip it if you're not in the mood for that.

> I _cannot believe_ that more companies, and especially startups, don't do this stuff. It is in the top five most impactful decisions we've made.

Reviewer: ```suggestion
I _cannot believe_ that more companies, and especially startups, aren't more transparent. It is in the top five most impactful decisions we've made.
```

As above.

Author: i think this loses some personality in the name of clarity, so skipping this one

> Above all, it enables people to work autonomously. There are two ways to grow a team - autonomy, or control. We chose the former because it's the kind of place we'd want to work, and therefore it makes it a better place to work for our team. That lets us hire and retain stronger people. Talent compounds, y'all.

Reviewer 2: ```suggestion
Above all, it empowers people to work autonomously. There are two ways to grow a team – autonomy, or control. We chose the former because it's the kind of place we'd want to work, and therefore it makes it a better place to work for our team. That lets us hire and retain stronger people. Talent compounds, y'all.
```

Author: empowers is more corporate than enables

Transparency works particularly well in an all remote company. Like ours. That's because having this stuff written down and shared makes asynchronous work much easier.

> We didn't expect this, but hiring great has become much easier. Since we share most of the below publicly, people thinking about working here can build trust much more easily, and it makes us stand apart from the thousands of other startups. Transparency makes the whole company a joint project, between our team and the community around it too.

Reviewer: ```suggestion
We didn't expect this, but transparency makes hiring great talent much easier. Since we share most of the below publicly, potential hires can trust us much more easily, and it makes us stand apart from the thousands of other startups. Transparency makes the whole company a joint project, between our team and the community around it too.
```
Eliminating some passive voice and adding a missing word.

> ## General communication

Reviewer: Main bit of feedback from me is that I don't really get this structure or subheadings from here onwards. General communication isn't something we're transparent about, for example, nor is it a useful bucket for some of the contained topics (e.g. tool minimalism). I don't think `People` is a good subheading either, as it makes the actual content of these sections (e.g. the actual list of what we're transparent about) harder to locate in the article. 

Personally, I'd just restructure some of this content so that this section is called 'What we're transparent about'. I'd use the topic of tool minimalism as an intro, then I'd list all of the things we're transparent about as directly as possible: Fundraising, Hiring/Firing Decisions, Pay Structure, etc.
